Services/Devices.cs

```diff
@@ -7,6 +7,7 @@
 using Microsoft.Azure.Devices;
```

Comment:
Will remove changes to this file.

Reply:
I assume there is no need to depend on rateLimiting, right?